gui/src/components/onboarding/pages/stay-aligned/stay-aligned-steps/VerifyMounting.tsx

```diff
@@ -53,6 +65,25 @@ export function VerifyMountingStep({
           <Button variant={'secondary'} onClick={prevStep}>
             {l10n.getString('onboarding-automatic_mounting-prev_step')}
           </Button>
+          <Button
+            variant={'secondary'}
+            className="self-start mt-auto"
+            onClick={setOpenFunc}
+          >
+            {l10n.getString('onboarding-automatic_mounting-manual_mounting')}
+          </Button>
+
+          <BaseModal isOpen={isOpen}>
+            <ManualMountingPageStayAlligned />
+
+            <Button
+              variant="primary"
+              className="-top-44 left-8"
+              onClick={setCloseFunc}
```

Comment:
use the id attribute instead of getString for translations keys. Both on buttons and typography